src/coreclr/inc/readytorun.h

```diff
@@ -121,6 +124,21 @@ enum class ReadyToRunImportSectionFlags : uint16_t
     PCode    = 0x0004, // Section contains pointers to code
 };
 
+enum class ReadyToRunTypeGenericInfo : uint8_t
```

Comment:
Is there a reason these are the values they are?

Reply:
Yes, by choosing these values we are able to store the data here in 4 bit chunks which pack nicely into 8 bit bytes.